playbooks/portals-deploy.yml

```diff
@@ -28,30 +36,25 @@
   gather_facts: False
 
   # Limit concurrency
-  serial: 1
+  serial: "{{ parallel_deploys | default(1) }}"
```

Comment:
Can we rename this var to `parallel`? `serial` sounds like a boolean flag while `parallel` sounds like something that controls how many parallel streams of deployments we have. Which functions the same way for `parallel: 1`.

Reply:
I don't think so. Fil can confirm but `serial` is an ansible builtin, not something we are defining.

Reply:
I also don't quite understand the interaction between the two `serial` vars. When would we want to set the first `serial` to something other than 1?

Reply:
> I don't think so. Fil can confirm but `serial` is an ansible builtin, not something we are defining.

Oh, that would make sense.

Reply:
`serial` is a built-in Ansible variable. Our name is `parallel_deploys`.